dask/tests/test_distributed.py

```diff
@@ -1,31 +1,26 @@
-from dask.highlevelgraph import HighLevelGraph, MaterializedLayer
 import pytest
 
+from dask.highlevelgraph import HighLevelGraph, MaterializedLayer
+
```

Comment:
This looks off, could you try moving it down below with the rest of the `dask` imports to see what happens?